SynthMoog.cpp

```diff
@@ -0,0 +1,49 @@
+#include <iostream>
+#include "SynthMoog.h"
+
+SynthMoog::SynthMoog()
+{
+
+}
+
+SynthMoog::~SynthMoog()
+{
+    std::cout << "Calling DTOR - SynthMoog" << std::endl; 
+}
+
+void SynthMoog::setVoiceFreq(unsigned int voiceNum, float freq)
+{
+    if ( voiceNum <=3 )
+    {
```

Comment:
add space around your operators so your code is more readable
`voiceNum <= 3`
- [ ] revise